packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -243,6 +248,7 @@ const OTPInput = ({
             errorText={errorText}
             helpText={helpText}
             hideFormHint={true}
+            type={isMasked ? 'password' : 'numeric'}
```

Comment:
Shouldn't this be `type="text"` and `inputmode="numeric"`? couldn't find any mdn documentation of `numeric` type as such https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#using_number_inputs

Reply:
@chaitanyadeorukhkar - https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Input/BaseInput/_decisions/_decisions.md#why-we-are-not-having-keyboardtypenumeric

Reply:
Yeah, seems like type `numeric` doesn't exist. I used `BaseInput`'s props only. We have specific `numeric` as an acceptable value for type. Let me remove this change. For now I'll just keep it as is and raise an issue for it.

Reply:
@kamleshchandnani the link you specified is for `keyboardType` attribute not for `type`. `type=numeric` doesn't exist but we accept that value. We should ideally make it `type=number`?

Reply:
Went through [this](https://css-tricks.com/better-form-inputs-for-better-mobile-user-experiences/)

So we don't want to pass `type=number` instead we want to pass `inputmode=decimal`. inputmode is `keyboardType` for us which we're already passing. `type=numeric` seems to be an incorrect value we're accepting and rather it should be `type=number`. I'll make that change in a different PR

<img width="462" alt="Screenshot 2022-12-01 at 4 34 34 PM" src="https://user-images.githubusercontent.com/24487274/205036952-d6327e5f-8169-40f0-86ef-02a99d0ca8c4.png">

Reply:
Raised a PR for that change